scene/resources/tile_set.cpp

```diff
@@ -907,6 +907,8 @@ void TileSet::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("tile_get_material", "id"), &TileSet::tile_get_material);
 	ClassDB::bind_method(D_METHOD("tile_set_texture_offset", "id", "texture_offset"), &TileSet::tile_set_texture_offset);
 	ClassDB::bind_method(D_METHOD("tile_get_texture_offset", "id"), &TileSet::tile_get_texture_offset);
+	ClassDB::bind_method(D_METHOD("tile_get_modulate", "id"), &TileSet::tile_get_modulate);
+	ClassDB::bind_method(D_METHOD("tile_set_modulate", "id"), &TileSet::tile_set_modulate);
```

Comment:
`color` arg?

Reply:
Oh, I missed that, sorry! Thanks for reviewing :) wasn't thinking of it as a feature but makes sense.